PresentScreenings/AppDelegate.cs

```diff
@@ -108,8 +108,8 @@ void ShowSaveAs(NSObject sender)
             {
                 string directory = dlg.Directory;
 
-                // Write film info.
-                FilmInfo.SaveFilmInfoAsXml(ScreeningsPlan.FilmInfos, Path.Combine(directory, "filminfo.xml"));
+                //// Write film info.
+                //FilmInfo.SaveFilmInfoAsXml(ScreeningsPlan.FilmInfos, Path.Combine(directory, "filminfo.xml"));
```

Comment:
I do prefer to keep this and related commented out code in order to recover the functionality as fast as possible when parsing IFFR 2021 film information with `python` would be to elaborate.
I will create a version switch to prevent commented out code.